packages/markdown-docx/src/OoxmlTransformer.js

```diff
@@ -151,6 +151,17 @@ class OoxmlTransformer {
                 $class: `${NS_PREFIX_CommonMarkModel}Softbreak`,
             };
         case 'w:r':
+            if(element.elements[0].name==='w:rPr'){
+                let emphFound = element.elements[0].elements.some(subElement=>{
+                    return subElement.name==='w:i' && subElement.attributes['w:val'] === 'true';
```

Comment:
> subElement.attributes['w:val'] === 'true';

This is not necessary. Italics is valid without it too.

Reply:
In fact, we would need to delete this attribute from the `CIceroMark -> OOXML` transformer too.